docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -207,6 +207,18 @@ quarkus.oidc.credentials.jwt.key-id=mykeyAlias
 
 Using `client_secret_jwt` or `private_key_jwt` authentication methods ensures that a client secret does not get sent to the OIDC provider, therefore avoiding the risk of a secret being intercepted by a  'man-in-the-middle' attack.
 
+.Example how JWT Bearer token can be used to authenticate client
+
+[source,properties]
+----
+quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus/
+quarkus.oidc.client-id=quarkus-app
+quarkus.oidc.credentials.jwt.source=bearer <1>
+quarkus.oidc.credentials.jwt.jwt-bearer-token-path=/var/run/secrets/tokens <2>
```

Comment:
```suggestion
quarkus.oidc.credentials.jwt.token-path=/var/run/secrets/tokens <2>
```

Sorry Michal, I  know we briefly discussed the property name and a `jwt-bearer-token-path` is totally fine, but once I looked at the complete config example, I thought there was a bit of duplication, both for `jwt` and `bearer` :-)

Reply:
I agree, the `token-path` sounds better. Thanks.